dfttopif/parsers/pwscf.py

```diff
@@ -381,3 +380,28 @@ def get_band_gap(self):
             else:
                 bandgap = float(top-bot)
                 return Property(scalars=round(bandgap,3), units='eV')
+
+    def get_one_electron_energy_contribution(self):
```

Comment:
Lots of copy-paste here. Can you make one method that does that last three lines of each of these methods and just pass the key into it?

Reply:
Done.  The cases that aren't covered are either when the value isn't a single scalar or when the return object is a `Value` instead of a `Property`